tests/fixtures.py

```diff
@@ -71,24 +231,84 @@ def project_live(auth_live):
 
 
 @pytest.fixture()
-def workflow_mock(auth_mock):
-    workflow_id = "workflow_id123"
-    with requests_mock.Mocker() as m:
-        url_workflow_info = (
-            f"{auth_mock._endpoint()}/projects/"
-            f"{auth_mock.project_id}/workflows/"
-            f"{workflow_id}"
-        )
-        m.get(
-            url=url_workflow_info,
-            json={"data": {"xyz": 789, "name": "workflow_name_123"}, "error": {}},
-        )
+def workflow_mock(auth_mock, requests_mock):
+    # info
+    url_workflow_info = (
+        f"{auth_mock._endpoint()}/projects/"
+        f"{auth_mock.project_id}/workflows/"
+        f"{WORKFLOW_ID}"
+    )
+    json_workflow_info = {
+        "data": {
+            "name": WORKFLOW_NAME,
+            "id": WORKFLOW_ID,
+            "xyz": 789,
+        },
+        "error": {},
+    }
+    requests_mock.get(url=url_workflow_info, json=json_workflow_info)
```

Comment:
Why is this preferable to using a context manager?

Reply:
With context manager the function call needs to be inside the context, wouldn't allow to put it in the fixture.